frontend/src/app/app.module.ts

```diff
@@ -99,6 +105,8 @@ const appRoutes: Routes = [
     IconComponent,
     IconLabelComponent,
     BannerComponent,
+    LoginComponent,
```

Comment:
`LoginComponent` is already mentioned here, 5 lines above

Reply:
Duplicated LoginComponent  was deleted